monkey/monkey_island/cc/ui/package.json

```diff
@@ -93,6 +93,7 @@
     "react-table": "^6.8.6",
     "react-toggle": "^4.0.1",
     "redux": "^4.0.0",
-    "sha3": "^2.0.0"
+    "sha3": "^2.0.0",
+    "react-spinners": "latest"
```

Comment:
This sounds like a bad idea

Reply:
Pinned the version to minor releases only (however, react-spinners is not yet in 1.0.0...)